_data/authors.yml

```diff
@@ -42,6 +42,9 @@ lian-yuanlin:
   name: Lian Yuanlin
   thumbnail: /img/authors/lian-yuanlin.jpg
   github: alienchow
+muchen-tang:
+  name: Muchen Tang
```

Comment:
I think we're still sticking to the Asian naming format here.

Reply:
using his full name in slack

Reply:
It's inconsistent if we have some `<first name> <last name>` and others `<last name> <first name>` for chinese names.

Reply:
but we have `Rachel Lee` `Ryan Law`?

Reply:
okay, asked author, he prefers `Tang Muchen` so I will change accordingly